test/global-table.test.ts

```diff
@@ -64,6 +111,12 @@ function testGrant(expectedActions: string[], invocation: (user: iam.IPrincipal,
     partitionKey: {
       name: 'id', type: ddb.AttributeType.STRING,
     },
+    replicas: [{
+      region: 'us-east-2',
+    },
+    {
+      region: 'us-west-1',
+    }],
```

Comment:
Does adding replicas affect the grant? Did you verify with the integration test lambdas that when you write to the table with replicas, the data shows up in all regions?

Reply:
What happens if the DynamoDB client in your integration tests is a replication region (not us-east-1)? Is it still able to write to the table?